modules/tinymce/src/themes/silver/main/ts/ui/core/color/Options.ts

```diff
@@ -82,11 +82,23 @@ const register = (editor: Editor): void => {
     processor: 'boolean',
     default: true
   });
+
+  registerOption('color_foreground_default', {
+    processor: 'string',
+    default: 'black'
```

Comment:
Similar to my other comment, I think this should be `#000000` instead. Also, if setting a default, the `EditorOptions` interface should be updated as well in [OptionTypes.ts](https://github.com/tinymce/tinymce/blob/45b889dd48b7433a1ef748aa1d4f844b82b0c91a/modules/tinymce/src/core/main/ts/api/OptionTypes.ts#L265)